app/src/main/java/io/neurolab/NeuroSettings.java

```diff
@@ -1,10 +1,13 @@
 package io.neurolab;
 
+
 public class NeuroSettings {
 
-    public int samplesPerSecond = 3;
+
+    //Default place holder values
+    public int samplesPerSecond = 4;
     public int numChannels = 2;
-    public int bins = 4;
+    public int bins = 3;
```

Comment:
Why incrementing all these?

Reply:
@CloudyPadmal these are just some default place holder values. And they are not getting incremented anywhere sir.

Reply:
I meant it was `samplesPerSecond = 3` and in this PR it was changed to `samplesPerSecond = 4`. I think this change is not needed.